gui/src/test/java/kcl/teamIndexZero/traffic/gui/GUI_PrimitivesTest.java

```diff
@@ -0,0 +1,48 @@
+package kcl.teamIndexZero.traffic.gui;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by kumarawijeet on 3/16/2016.
+ */
+public class GUI_PrimitivesTest {
```

Comment:
Testing that your exceptions are thrown in the right circumstances (i.e. when the file is either corrupted or absent) would be a good thing to do here I would think.